JsonSchema/JsonSchema.cs

```diff
@@ -814,7 +810,11 @@ public override void Write(Utf8JsonWriter writer, JsonSchema value, JsonSerializ
 		foreach (var keyword in value.Keywords!)
 		{
 			writer.WritePropertyName(keyword.Keyword());
-			options.Write(writer, keyword, keyword.GetType());
+
+			var keywordType = keyword.GetType();
+			var typeInfo = SchemaKeywordRegistry.GetTypeInfo(keywordType);
+			var converter = (IWeaklyTypedJsonConverter)typeInfo.Converter;
```

Comment:
This is _slightly_ asymmetric with the read. options.Read handles the Converter not implementing IWeaklyTypedJsonConverter, but the Write doesn't. Might want to leverage the same kind of helper here.